### DISCUSSION TITLE
Why does not augmentation increase the number of training data?

### DISCUSSION THREAD

**Discussion by elec-web** [NVIDIA-NeMo/NeMo#1784] [Q&A] [answered]
I have used vad configure file including augmenor with shift and white noise perturbation, spec_augment but when I try to apply it in train phase with 1000 training samples, the number of traning samples is fixed and it does not increase by augmentation. Why does not my training samples with these augmentations increase?
Thanks in advance.

**Comment by nithinraok** [comment, marked as answer]
Perturbation augmentation in NeMo is applied if probability at random sample > prob of augmentation selected. So total num of samples (iterations) remains the same. I suggest you train for a higher number of epochs for effective usage of augmentation.
